admin/users/views.py

```diff
@@ -0,0 +1,32 @@
+from django.shortcuts import redirect
+
+from website.project.model import User
+
+from admin.base.views import GuidFormView, GuidView
+from admin.users.templatetags.user_extras import reverse_user
+from .serializers import serialize_user
+
+
+def remove_2_factor(request, guid):
+    if request.method == 'POST' or request.method == 'DELETE':
```

Comment:
This conditional isn't necessary unless functionality from ADM-79 is implemented.